### ISSUE TITLE
SyntaxError: invalid syntax

### ISSUE THREAD

**Issue by LaurentGatto** [rstojnic/lazydata#1]
Using Python 2.7.6 and the example script you provide, with a file outside of the github repo (file exists and file path is correct, I checked):
```python
from lazydata import track

with open(track("/home/lg390/tmp/data/some_data_file.txt"), "r") as f:
    print(f.read())
```
I get
```
-> % python sample_script.py
Traceback (most recent call last):
  File "sample_script.py", line 1, in <module>
    from lazydata import track
  File "/usr/local/lib/python2.7/dist-packages/lazydata/__init__.py", line 1, in <module>
    from .tracker import track
  File "/usr/local/lib/python2.7/dist-packages/lazydata/tracker.py", line 11
    def track(path:str) -> str:
                  ^
SyntaxError: invalid syntax
```

**Comment by rstojnic** [comment]
Thanks! Currently only Python 3.5+ is supported - will add this to the docs.

**Comment by varun19299** [comment]
Could you add this to your pip dependencies? Ensuring its not used with < 3.5?

**Comment by rstojnic** [comment]
Released `1.0.18` that requires python >= 3.5.2
